### ISSUE TITLE
Use `undefined` instead of `null` for `aria-label` and `aria-hidden` attributes

### ISSUE THREAD

**Issue by bdbaraban** [SeanMcP/a11y-react-emoji#4]
When no `label` prop is specified, the type of `aria-label` should be `undefined` instead of `null`. Same goes for `aria-hidden` when a `label` is passed. This aligns with the official React type definitions for both attributes.

Type of `aria-label`:
```
(JSX attribute) React.AriaAttributes['aria-label']?: string | undefined
```

Type of `aria-hidden`:
```
(JSX attribute) React.AriaAttributes['aria-hidden']?: boolean | "true" | "false" | undefined
```
